src/hardware/mixer.cpp

```diff
@@ -295,45 +289,30 @@ bool MIXER_FastForwardModeEnabled()
 // Individual channels also have a mutex which can be safely aquired without stopping these queues
 void MIXER_LockMixerThread()
 {
-	if (pc_speaker) {
-		pc_speaker->output_queue.Stop();
-	}
-	if (tandy_dac) {
-		tandy_dac->output_queue.Stop();
-	}
-	if (ps1_dac) {
-		ps1_dac->output_queue.Stop();
-	}
-	if (lpt_dac) {
-		lpt_dac->output_queue.Stop();
-	}
```

Comment:
So we don't need similar changes for the LPT DAC devices?

Reply:
I don't think so; from my tests, they're working fine as-is.